### ISSUE TITLE
Neon.2 does not show workspace label

### ISSUE THREAD

**Issue by turesheim** [turesheim/eclipse-launcher#5281]

**Comment by turesheim** [comment]
Oxygen comes with Oomph badge management installed both in installation created by the installer and the package. One has to manually change the workspace name in order for the badge to show. This is regardless of this plug-in being installed or not.

**Comment by asafd1** [comment]
so I changed the names of my workspaces (i have 2) and opened the first one and it was tagged correctly. then I opened the second one and while it was launching it was tagged correctly. but when it finished launching both badges disappeared :-( 
eclipse: Neon.2 Release (4.6.2)
osx: el capitan 10.11.6 (15G19009) 
OS X Eclipse Launcher Utility: 3.0.0.201509271854

**Comment by turesheim** [comment]
Yes, I've seen this and I believe it is due to some timing issues related to Oomph (coming with most Eclipse installations now) interfering with the label. I believe it can be resolved if you manually change the workspace name.

**Comment by asafd1** [comment]
if you mean to change the workspace forlder on the file system, then it doesn't help. the same thing happens:
when I open the first workspace it was tagged correctly. then I opened the second workspace and now non of them is tagged

**Comment by turesheim** [comment]
No. You should open **Preferences > Workspace**  and look for **Window Title > Show workspace name**. There is no need to check the box, but you should enter a name for the workspace. This will then be used as the icon badge.

**Comment by asafd1** [comment]
that did it. thanks!
